### PR TITLE
Partners page updates

### PR DESCRIPTION
**Pull request by kbarnard10**
New listings for kubernetes.io/partners

### CODE REVIEW

**Comment by kbarnard10** [comment]
cc // @zacharysarah

File: _includes/partner-script.js [hunk 1]
```diff
@@ -1344,13 +1351,104 @@
...
+			name: 'Baidu',
+			logo: 'baidu',
+			link: 'https://cloud.baidu.com/product/cce.html',
+			blurb: 'Baidu Cloud - Baidu Cloud Container Engine'
+				},
 		{
 			type: 3,
 			name: 'ZTE',
 			logo: 'zte',
 			link: 'https://sdnfv.zte.com.cn/zh-CN/home',
```
Comment (xiangpengzhao) [inline _includes/partner-script.js]: As a ZTEr, I'd like here to use this link https://sdnfv.zte.com.cn/en/home for i18n :)

**Comment by Bradamant3** [comment]
@kbarnard10 is this for 1.10? If so, can you please rebase against the `release-1.10` branch? If that's problematic, we can hold off on merging until release day.

/assign

**Comment by kbarnard10** [comment]
@Bradamant3 this is independent of the 1.10 release and can be merged at any time.

File: _includes/partner-script.js (hunk 1 above)
Comment (kbarnard10) [reply _includes/partner-script.js]: Updated - thanks!

**Pull request merged**